charts/ccsm-helm/charts/zeebe-gateway/templates/configmap.yaml

```diff
@@ -0,0 +1,41 @@
+kind: ConfigMap
+metadata:
+  name: {{ include "zeebe-gateway.fullname" . }}
+  labels: 
+    {{- include "zeebe.labels.gateway" . | nindent 4 }}
+apiVersion: v1
+data:
+  startup.sh: |
+    #!/usr/bin/env bash
+    set -eux -o pipefail
+
+    export ZEEBE_BROKER_NETWORK_ADVERTISEDHOST=${ZEEBE_BROKER_NETWORK_ADVERTISEDHOST:-$(hostname -f)}
```

Comment:
will you rework this file later? Not sure if these configs are useful for the gateway?

Reply:
Yes I will work on this. Currently it is the same as before. The Standalone gateway before used the same configmap as the broker. :)

Reply:
okay